cli/internal/cache/cache.go

```diff
@@ -271,6 +277,21 @@ func (mplex *cacheMultiplexer) Fetch(target string, key string, files []string)
 	return false, files, 0, nil
 }
 
+func (mplex *cacheMultiplexer) Exists(target string) (CacheState, error) {
+	syncCacheState := CacheState{}
+	for _, cache := range mplex.caches {
+		cacheState, _ := cache.Exists(target)
```

Comment:
I don't think we should ignore the error here, especially since this function is equipped to return an error already.